### PR TITLE
test: move e2e deployment test in extended suite

### PR DESCRIPTION
**Pull request by kargakis**
Closes https://github.com/openshift/origin/issues/8078

@mfojtik

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

**Comment by mfojtik** [comment]
LGTM (after test pass)

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -246,6 +247,29 @@ var _ = g.Describe("deploymentconfigs", func() {
 		})
 	})
 
+	g.Describe("when tagging images", func() {
+		g.AfterEach(func() {
+			failureTrap(oc, "tag-images", g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should successfully tag the deployed image [Conformance]", func() {
```
Comment (smarterclayton) [inline test/extended/deployments/deployments.go]: Needs to do this for a test deployment - that's an important part of this test.

Comment (smarterclayton) [inline test/extended/deployments/deployments.go]: Unless this was the older test case - the intent was to test the tagging with a test deployment (even if this actually wasn't testing it before)

Comment (kargakis) [inline test/extended/deployments/deployments.go]: Updated the fixture to be a test deployment, even though it doesn't have to do anything with tagging.

Comment (smarterclayton) [inline test/extended/deployments/deployments.go]: Use case driven - few people should use tags with regular deployments.

Comment (kargakis) [inline test/extended/deployments/deployments.go]: I see - we need to surface this scenario of promoting images in our docs

**Comment by kargakis** [comment]
[merge]

**Pull request merged**